server.c

```diff
@@ -261,7 +302,20 @@ static int fio_recv_data(int sk, void *buf, unsigned int len, bool wait)
 		flags = OS_MSG_DONTWAIT;
 
 	do {
-		int ret = recv(sk, p, len, flags);
+		int ret;
+#ifdef _WIN32
```

Comment:
This should not be win32 specific, I'd suggest just making it so in general.

Reply:
I ended up having to revert this change and move to a more client-specific change in fio_drain_client_text().  I did update the final change so that it is now for all builds.